xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd

```diff
@@ -407,6 +407,16 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="vehicleLoadingPlace"/>
 			<xsd:enumeration value="handicappedLevelBoardingSector"/>
 			<xsd:enumeration value="multimodal"/>
+			<xsd:enumeration value="busOnStreet">
+				<xsd:annotation>
+					<xsd:documentation>bus stop directly on street.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="tramOnStreet">
```

Comment:
Same as for bus.... we have already tramStop and TramPlatform .... I was expecting tramStop to be on street.

Reply:
We refer to a situation where a bus or tram stops in the middle of the street, but there is no separate quay and/or crossing equipment. This means that passengers have to cross a lane in traffic if they want to board the vehicle. Maybe an enum name like "busOnStreetCentre" would express it more clearly.

Reply:
Ok, it was not obvious at first: it needs to be clarified so people won't use one instead of the other (and renamed_centerStreed_ instead of _onStreet_, or the _OnStreetCentre_ you proposed )

Reply:
>   Is the essence here that boarding and alighting the vehicle take place directly on the street? If yes, then I suggest to use "busOnStreetBoarding". "tramOnStreet" is clear. It's hard for me to imagine what this would look like for a bus. For tram there is a priority rule: when a tram stops at a "tramOnStreet", the tram may not be overtaken on the right so that passengers can get on and off safely. For busses we don't have onstreetboarding in the Netherlands.

Reply:
I have seen the following situations:
Bus stops at a place where there is no platform or pedestrian lane on the right side - passengers have to cross the road (either before or after a,rrival of the bus) for boarding. (I think this is illegal in the long run, but may exist for several years.) Almost like Joost's proposal, I would call it busBoardingOnStreet

tramBoardingOnStreet - a tram stop (which may even have a narrow platform), but on the right of the stop there is a bus lane where buses and taxis (and probably bicycles) may pass. This riskier situation might be labellled tramBoardingOnStreet (unless, perhaps, if there is a road crossing nearby - then it might be considered a regular stop).

Reply:
changed names